src/_listNavigationItem.scss

```diff
@@ -0,0 +1,352 @@
+
+        width: auto;
+        display: block;
+      }
+
+      .#{$block}__navigation-item-indicator {
+        &::after {
+          @include set-margin-right();
+        }
+      }
+    }
+
+    &.is-expanded > .#{$block}__navigation-item-indicator {
+      &::after {
+        display: none;
+      }
+    }
+
+    &--indicated {
+      color: var(--sapSelectedColor);
+
+      .#{$block}__navigation-item-text,
+      .#{$block}__navigation-item-icon,
+      .#{$block}__navigation-item-arrow {
+        color: var(--sapSelectedColor);
+      }
+    }
+
+    .#{$block}__navigation-item--indicated .#{$block}__navigation-item-text {
+      color: var(--sapSelectedColor);
+    }
+
+    &--condensed {
+      @include set-paddings-x();
+
+      position: relative;
+      width: 3.25rem;
+
+      .#{$block}__navigation-item-popover {
+        &--first-level,
+        &--second-level {
+          display: none;
+          background: var(--sapList_AlternatingBackground);
+        }
+      }
+
+      .#{$block}__navigation-item-text {
+        display: none;
+      }
+
+      .#{$block}__navigation-item-icon {
+        @include set-margins-x(0.5rem, 0.5rem);
+
+        height: $fd-list-navigation-item-height;
+        line-height: $fd-list-navigation-item-height;
+      }
+
+      &.is-expanded {
+        @include set-margins-x();
+
+        overflow: visible;
+        visibility: hidden;
+
+        .#{$block}__navigation-item {
+          @include set-margins-x();
+        }
+
+        .#{$block}__navigation-item--condensed {
+          .#{$block}__navigation-item-icon {
+            margin-left: 0;
+            padding-left: 0.875rem;
+
+            @include fd-rtl() {
+              margin-right: 0;
+              margin-left: 0.5rem;
+              padding-right: 0.875rem;
+            }
+          }
+        }
+
+        .#{$block}__navigation-item-popover {
+          &--first-level,
+          &--second-level {
+            overflow: hidden;
+            display: block;
+
+            .fd-list {
+              width: 100%;
+              display: inline-block;
+              margin: 0 auto;
+            }
+
+            .#{$block}__navigation-item-text {
+              @include set-margin-left();
+              @include set-padding-left();
+            }
+          }
+
+          &--first-level {
+            top: 0;
+            left: 0.325rem;
+            width: 14.325rem;
+            border-bottom-right-radius: 0;
+            z-index: 3;
+            height: 100%;
+
+            @include fd-rtl() {
+              border-bottom-left-radius: 0;
+              border-bottom-right-radius: var(--sapElement_BorderCornerRadius);
+              left: unset;
+              right: 0.325rem;
+            }
+          }
+
+          &--second-level {
+            border-top-right-radius: 0;
+            border-top-left-radius: 0;
+            top: $fd-list-navigation-item-height;
+            left: 3.25rem;
```

Comment:
I would move this repeated value to separate variable

Reply:
done